src/lsp/server.ts

```diff
@@ -24,7 +24,7 @@ export function languageServerCommand(
 
   const params = [];
   if (DEBUG) {
-    params.push('-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8001,quiet=y');
+    params.push('-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8000,quiet=y');
```

Comment:
Why change the port?

Reply:
I accidentally changed the port to 8001 in a [previous commit](https://github.com/SonarSource/sonarlint-vscode/commit/ac87f1a74ee2ce75f36e374b76bc3d75785d3bdb#diff-871030d68031ea7ae229b410ff486fee8a1b1e637d56924d6376d1747bb87a72). This is just to revert the change

Reply:
It's a change not related to the ticket. Can you please put it as a separate commit into this PR and don't squash them on merge? So history would be clear.